src/stylesheets/elements/form-controls/_date-picker.scss

```
@@ -1,5 +1,50 @@
 $date-picker-max-width: "mobile";
 
+// Maps for high contrast mode icons
+
+$calendar-icon-defaults: (
+  color: "ButtonText",
+  svg-height: 20,
+  svg-width: 20,
+  height: units(5),
+  container-width: 6,
```

@mahoneycm  I believe it is best practice to quote the keys inside sass maps. It looks like we follow this practice elsewhere in the system

```suggestion
  "color": "ButtonText",
  "svg-height": 20,
  "svg-width": 20,
  "height": units(5),
  "container-width": 6,
```

Reference [Sass Maps page:](https://sass-lang.com/documentation/values/maps)
"Most of the time, it’s a good idea to use [quoted strings](https://sass-lang.com/documentation/values/strings#quoted) rather than [unquoted strings](https://sass-lang.com/documentation/values/strings#unquoted) for map keys. This is because some values, such as color names, may look like unquoted strings but actually be other types. To avoid confusing problems down the line, just use quotes!"

---

@mejiaj Are you ok with that? Earlier you had me remove the quotes but I don't mind adding them back if it helps to eliminate confusion down the line!